cinema/views.py

```diff
@@ -167,6 +188,32 @@ def get_serializer_class(self):
 
         return MovieSessionSerializer
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                name="title",
+                type=str,
+                description="Filter movies by title (case-insensitive)",
+            ),
+            OpenApiParameter(
+                name="genres",
+                type={"type": "array", "items": {"type": "integer"}},
+                description="""Filter movies by genre IDs.
+                Use comma-separated values
+                (e.g., 1,2,3)""",
+            ),
+            OpenApiParameter(
+                name="actors",
+                type={"type": "array", "items": {"type": "integer"}},
+                description="""Filter movies by actor IDs.
+                Use comma-separated values
+                (e.g., 1,2,3)""",
+            ),
+        ]
+    )
```

Comment:
The `extend_schema` decorator for the `list` method in `MovieSessionViewSet` includes parameters like 'title', 'genres', and 'actors', which are not used in the `get_queryset` method. Ensure that the documented parameters match the actual query parameters used in the method.